aries-site/src/layouts/content/Subsection.js

```diff
@@ -22,22 +29,50 @@ export const Subsection = ({ children, name, ...rest }) => {
       onFocus={() => setOver(true)}
       onBlur={() => setOver(false)}
     >
-      <Box direction="row" justify="between">
-        <Subheading {...rest}>{name}</Subheading>
-        <Anchor
-          a11yTitle={`Jump to section titled ${name}`}
-          href={`#${id}`}
-          icon={<LinkIcon color={over ? 'text-xweak' : 'transparent'} />}
-        />
+      {/* This condition for gap is needed because the link icon has padding
+       * to maintain a big enough touch area for accessibility. This creates
+       * a larger than desired gap between h3's and its first child. This
+       * removes that extra space.
+       */}
+      <Box gap={GAP_SIZE[level]}>
+        <Box direction="row" justify="between" align="center">
+          <Subheading level={level}>{name}</Subheading>
+          {level > 1 && (
+            <Anchor
+              a11yTitle={`Jump to section titled ${name}`}
+              href={`#${id}`}
+              icon={<LinkIcon color={over ? 'text-xweak' : 'transparent'} />}
+            />
+          )}
+        </Box>
+        {React.Children.map(children, (child, index) => {
+          if (index === 0) {
+            return React.cloneElement(child, {
+              level,
+            });
+          }
+          return undefined;
+        })}
       </Box>
-      {children}
+      {React.Children.map(children, (child, index) => {
+        if (index === 0) {
+          return undefined;
+        }
+        return React.cloneElement(child, {
+          level,
+        });
+      })}
     </Box>
   );
 };
 
 Subsection.propTypes = {
   children: PropTypes.oneOfType([PropTypes.element, PropTypes.array])
     .isRequired,
-  headingSize: PropTypes.string,
+  level: PropTypes.number,
   name: PropTypes.string.isRequired,
 };
+
+Subsection.defaultProps = {
+  level: 2,
```

Comment:
When refactoring to apply to MainHeading and MainDescription, will this then default to 1?

Reply:
See my above response on BodyText